xs2a-impl/src/main/java/META-INF/.gitignore

```diff
@@ -0,0 +1 @@
+/MANIFEST.MF
```

Comment:
Something, it could also be certain IDEs automatically creates `/aspsp-mock-server/src/main/java/META-INF/MANIFEST.MF`. I removed the entire META-INF folder but added .gitignore on a higher level.